lib/editor/components/pattern/PatternStopCard.js

```diff
@@ -68,6 +68,10 @@ type State = {
 }
 
 const pickupDropoffOptions = [
+  {
+    value: '',
+    text: '(default)'
```

Comment:
Sort attributes.

Reply:
Updated in [8f9dc88](https://github.com/ibi-group/datatools-ui/pull/902/commits/8f9dc88408c8b43db8ef249b85dde1ddfe753cf6)